ToolBox/KaikoDataConverter/Program.cs

```diff
@@ -87,7 +96,7 @@ private static void AggregateTicksInAllResolutions(string market, TickType tickT
                 var symbolDirectoryInfo = new DirectoryInfo(tickDirectory);
                 var symbol = Symbol.Create(symbolDirectoryInfo.Name, SecurityType.Crypto, market);
 
-                foreach (var tickDateFile in Directory.EnumerateFiles(symbolDirectoryInfo.FullName))
+                foreach (var tickDateFile in Directory.EnumerateFiles(symbolDirectoryInfo.FullName, "*.zip"))
```

Comment:
This can still fail too; please also try catch inside this foreach -- e.g. corrupt zip.